backend/src/main/java/com/elshop/backend/product/model/Product.java

```diff
@@ -0,0 +1,19 @@
+package com.elshop.backend.product.model;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.List;
+
+
+@Document(collection = "products")
+public record Product(
+        @Id String id,
+        String slug,
+        String title,
+        Double price,
```

Comment:
Bitte unbedingt ändern:
```suggestion
        BigDecimal price,
```